### PR TITLE
Revise ticket login requirements

### PR DESCRIPTION
**Pull request by barryhughes**
Updates to the way we handle RSVP/ticket purchase login requirements.

https://central.tri.be/issues/44347#note-22

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | You must have a space after a closing parenthesis | [rsvp.php:143](https://github.com/moderntribe/event-tickets/blob/3afe9f532f90633e7ddf3e1871fc6b128ebae7f2/src/views/tickets/rsvp.php#L143)

> via jenkins/codesniffer-phpcs

File: src/views/tickets/rsvp.php [hunk 1]
```diff
@@ -4,6 +4,7 @@
  *
  * @version 4.1
```
Comment (borkweb) [inline src/views/tickets/rsvp.php]: Change this to `@version 4.2`?

File: src/Tribe/Tickets.php [hunk 2]
```diff
@@ -1371,85 +1355,39 @@ public function front_end_tickets_form_in_content( $content ) {
...
-		 * Generally this will return true, but there may be special occasions such
-		 * as access being denied to logged out users and similar where it returns
-		 * false.
-		 *
-		 * @return bool
+		 * @return string
 		 */
-		protected function form_is_enabled() {
-			$enabled = true;
+		public static function get_login_link() {
```
Comment (borkweb) [inline src/Tribe/Tickets.php]: I wonder if this method should be named `get_login_url()` instead? It returns a URL rather than a link.

Comment (barryhughes) [inline src/Tribe/Tickets.php]: Good call.

File: src/views/tickets/rsvp.php (hunk 1 above)
Comment (barryhughes) [inline src/views/tickets/rsvp.php]: Good call!

**Comment by barryhughes** [comment]
Updated per review.

**Comment by borkweb** [comment]
:shipit:

**Pull request merged**